parallaxinc/solo
# Solo 332 terminal prevent cursor position from going negative

Addresses #332 
This error happened when the cursor was already at position 0,0, and the backspace key was pressed.  It caused the y position to go negative, and there is no such thing as a negative element in an array, so...typeError.

To test:
- open a console/terminal in blocklyProp
- click into the terminal to type
- make sure cursor is top-most and left-most
- press the backspace key
- verify that there are no longer any typeErrors in the console
